site/runDataTokens.tsx

```diff
@@ -2,13 +2,30 @@ import React from "react"
 import ReactDOM from "react-dom"
 import { DataToken_name } from "../site/DataToken.js"
 import { LastUpdated } from "./covid/LastUpdated.js"
+import { FullWidthRawHtml } from "./covid/FullWidthRawHtml.js"
 
 interface ComponentDictionary {
-    [key: string]: any
+    [key: string]: {
+        component: any
+        wrapper: DataTokenWrapper
+    }
 }
 
-const dictionary: ComponentDictionary = {
-    LastUpdated,
+type DataTokenWrapper = (children: React.ReactElement) => React.ReactElement
+
+export const dictionary: ComponentDictionary = {
```

Comment:
Late review here: I think `dictionary` is a bit too generic, maybe go with `componentDictionary` instead?